src/components/Alert.tsx

```diff
@@ -10,7 +10,7 @@ export interface AlertProps {
 export function Alert(props: AlertProps) {
   const { children, onClose } = props;
   return (
-    <div css={Css.df.itemsStart.justifyBetween.bgRed100.ba.bRed400.red700.p2.br4.$} role="alert">
+    <div css={Css.df.aifs.jcsb.bgRed100.ba.bRed400.red700.p2.br4.$} role="alert">
```

Comment:
I think these abbreviations will help people unfamiliar with the css definitions learn the full terms. For instance, just saying `justifyBetween` isnt' super clear that it becomes `justify-content: space-between`. I think `jcsb` will make people aware of the true output in the actual css rules.

Reply:
Agreed